v1/tests/__init__.py

```diff
@@ -0,0 +1 @@
+ECHO is on.
```

Comment:
What's this for?

Reply:
I'll delete that on windows how I create a file from the command line it comes with that